src/filters.js

```diff
@@ -82,18 +82,17 @@ const makeBetterSafe = ({ markAsSafe }) => {
 };
 
 const makeBuildStaticAddonCards =
-  ({ _buildStaticAddonCard = buildStaticAddonCard } = {}) =>
+  ({ _buildStaticAddonCard = buildStaticAddonCard, baseURL } = {}) =>
   async (value, callback) => {
-    const regexp = /<div class="addon-card" data-addon-id="(\d+)"><\/div>/g;
-
+    const regexp = /<div class="addon-card" data-addon-id="(\S+)"><\/div>/g;
```

Comment:
Using negate `\s` (so anything but (white)spaces) does not sound like a great idea because it isn't very readable. I'd suggest either `.+?` (non-greedy) or alphanumeric + `-` but not sure that is necessary.